### PR TITLE
update-86

### PR DESCRIPTION
**Pull request by stringhandler**
Esmeralda testnet

### CODE REVIEW

**Review by sdbondi — APPROVED** [review]

File: _updates/2022-08-11-update-86.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+## New testnet launching: Esmeralda
+
+The `dibbler` testnet has been running since January 2022, but there are new features in the development branch that are not in `dibbler`. 
+Because adding these features to `dibbler` would have required a development heavy hardfork, and we've already successfully tested hardforks in `dibbler`, 
+the development community decided to save time and create a new testnet instead. 
+
+Those features are now significant enough to start testing, and we're proud to annouce a new testnet [Esmeralda](https://en.wikipedia.org/wiki/Granny_Weatherwax). For those of you
+still mining, **Dibbler will continue to run, side-by-side with Esmeralda**. There's still two lambo's up for grabs at the [Tari Testnet Limited store](https://store.tarilabs.com/), which will continue to use `dibbler` tXTR.
```
Comment (CjS77) [inline _updates/2022-08-11-update-86.md]:
```suggestion
still mining, **Dibbler will continue to run, side-by-side with Esmeralda**. There are still two Lambos up for grabs in the [Tari Testnet Limited store](https://store.tarilabs.com/), which will continue to use `dibbler` tXTR. 
```

File: _updates/2022-08-11-update-86.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Because adding these features to `dibbler` would have required a development heavy hardfork, and we've already successfully tested hardforks in `dibbler`, 
+the development community decided to save time and create a new testnet instead. 
+
+Those features are now significant enough to start testing, and we're proud to annouce a new testnet [Esmeralda](https://en.wikipedia.org/wiki/Granny_Weatherwax). For those of you
+still mining, **Dibbler will continue to run, side-by-side with Esmeralda**. There's still two lambo's up for grabs at the [Tari Testnet Limited store](https://store.tarilabs.com/), which will continue to use `dibbler` tXTR. 
+
+> NOTE: `Esmeralda` testnet will most likely be reset a number of times as we add features, so if you are looking for stability, rather use `dibbler`. 
+
+Some of the new features in `Esmeralda`:
+* [Bulletproofs+](2022-06-24-update-80)
```
Comment (CjS77) [inline _updates/2022-08-11-update-86.md]:
```suggestion
* [Bulletproofs+](2022-06-24-update-80) - Bulletproofs, but better. Smaller than regular BPs, and with batch verification of range proofs, full-block validation times are something like 60% quicker. 😮 
```

File: _updates/2022-08-11-update-86.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+* [Stealth addresses](2022-08-01-update-84)
```
Comment (CjS77) [inline _updates/2022-08-11-update-86.md]:
```suggestion
* [Stealth addresses](2022-08-01-update-84) - Tari brought one-sided payments to Mimblewimble. Stealth addresses make the privacy trade-off much smaller. 🕶️ 
```

File: _updates/2022-08-11-update-86.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+* The ability to burn Tari (eventually this will form part of the peg in) [RFC](https://github.com/tari-project/rfcs/pull/10)
```
Comment (CjS77) [inline _updates/2022-08-11-update-86.md]:
```suggestion
* Tari 🔥  burn 🔥  transactions. Why would you want to do this?  [Find out more](https://github.com/tari-project/rfcs/pull/10)
```

**Review by CjS77 — COMMENTED** [review]
Can you add some more details?

1. When Esme is going live.
2. How to switch nodes and miners to / from Dibbler

**Review by CjS77 — APPROVED** [review]

**Pull request merged**